Source/UI/About/MessageButtonStyle.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  MessageButtonStyle.swift
+//  Planetary
+//
+//  Created by Martin Dutra on 9/12/22.
+//  Copyright © 2022 Verse Communications Inc. All rights reserved.
+//
+
+import SwiftUI
+
+struct MessageButtonStyle: ButtonStyle {
```

Comment:
Can you add a comment explaining what the `MessageButtonStyle` is, especially since there isn't a preview here?

Reply:
👌